### ISSUE TITLE
"Pending Scrobbles" Despite Not Pending

### ISSUE THREAD

**Issue by UserNo84** [kawaiiDango/pano-scrobbler#342]
Hi!

Since yesterday, the "pending scrobbles" persist even after submitting them and seeing them on Last.fm. By the way, on tablets, the cover art gets cropped at the top.

![Screenshot_20230610-015325_Pano Scrobbler](https://github.com/kawaiiDango/pScrobbler/assets/119287715/10e2baf5-cdcb-439e-b8be-f50c94ddb10c)

**Comment by kawaiiDango** [comment]
Are you using libre.fm? The site was down yesterday.

**Comment by UserNo84** [comment]
> Are you using libre.fm? The site was down yesterday.

Yes, I do. My Pano Scrobbler is connected to Last.fm, Libre.fm, and ListenBrainz.

**Comment by alexblonded** [comment]
Getting the same problem. Half of my scrobbles are lost. I've noticed it since this week, but may have been happening for more. Using lastfm
